exporter/streams.go

```diff
@@ -65,7 +75,9 @@ func getStreamInfo(c redis.Conn, key string) (*streamInfo, error) {
 }
 
 func getStreamEntryId(redisValue []interface{}, index int) string {
-	if len(redisValue) < index || redisValue[index] == nil || len(redisValue[index].([]interface{})) < 2 {
+	if _, ok := redisValue[index].([]interface{}); !ok ||
```

Comment:
This is getting a little crowded. 

I *think* this one is a direct prerequisite so that the last condition `len(redisValue[index].([]interface{}))` will not fail, right? Can we group them together?

Reply:
You means `if _, ok := redisValue[index].([]interface{}); !ok || len(redisValue) < index || redisValue[index] == nil || len(redisValue[index].([]interface{})) < 2` ?

Reply:
Sorry, I mistakenly used `strings.Contains` to determine `entry-id`, so it matched `max-deleted-entry-id`, which value cannot be asserted as `[]interface{}`.

Reply:
I like the second grouping, let's do that. And agreed, we should try to prevent the panic either way.

Reply:
Has been modified.